config/default.json

```diff
@@ -71,6 +71,9 @@
   "http": {
     "timeout": 10000
   },
+  "requestRetry": {
+    "maxAttempts": 5
```

Comment:
I would say 3 would be enough. Retrying 5 times on most HTTP errors (status code >= 300) seems pointless.